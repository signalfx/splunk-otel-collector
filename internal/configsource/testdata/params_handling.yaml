single_line:
  ex0: ${tstcfgsrc:elem0}
  ex1: ${tstcfgsrc:elem1?p0=true&p1=a string with spaces&p3=42}
