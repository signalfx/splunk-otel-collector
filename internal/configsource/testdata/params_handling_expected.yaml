single_line:
  ex0:
  ex1:
    p0: true
    p1: a string with spaces
    p3: 42
